libraries/botbuilder-ai/src/luis-client/luisClient.ts

```diff
@@ -51,11 +51,23 @@ export interface PredictionResolveOptionalParams {
     customHeaders: {headers: {[name: string]: string}};
 }
 
+ /**
+  *  Sets headers for request to LUIS service.
+  *  LUIS services use the `Ocp-Apim-Subscription-Key` header 
+  */
 export enum LuisApikeys {
     apiKeyHeader,
 }
 
+/**
+ * Luis Client Class
+ *
+ * @remarks
+ * This class is helper class for all the http request operations against the LUIS API
```

Comment:
I would say: `This is a helper class for...` or 'This class is a helper for...'

Reply:
Thank you, fixed.